CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` would be raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+- `Session.create_dataframe()` no longer ignores the optional parameter `schema` when `data` is a pandas DataFrame.
+
+### New Features:
+- Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` in functions.
+- Enabled creating UDFs that reference non-local functions. This feature is not available inside stored procedures.
+- Added the property `DataFrame.dtypes` to return a list of column name, data type pairs.
+- Added the following aliases:
+  - `functions.expr()` for `functions.sql_expr()`.
+  - `functions.date_format()` for `functions.to_date()`.
+  - `functions.monotonically_increasing_id()` for `functions.seq8()`
+  - `functions.from_unixtime()` for `functions.to_timestamp()`
+
+### Bug Fixes:
+- Fixed a bug in SQL simplifier that didn’t handle Column alias and join well in some cases.
```

Comment:
I suggest to state explicitly which cases were changed. What was the bug about?

Reply:
This is from #649 . It's kinda complicated since the fix enables a workaround but does not entirely solve the diamond-shaped join problem, which requires a separate design process. 
cc: @sfc-gh-yixie Please correct me if I'm wrong.

Reply:
Yes it's not easy to explain the bug. How about creating a github issue with detailed description and put the issue link in the release notes?

Reply:
That works for me.